x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/bytes/index.test.tsx

```diff
@@ -26,6 +26,8 @@ describe('Bytes', () => {
           contextId="test"
           eventId="abc"
           fieldName="network.bytes"
+          fieldType="string"
+          isAggregatable={true}
           isDraggable={true}
           value={`1234567`}
```

Comment:
Yeah, we should really have a lint rule for this :D Is there some sort of styleguide for Kibana?